src/js/stage.js

```diff
@@ -213,8 +213,11 @@ define(["d3", "palette", "transitions/default"], function (d3, palette, defaults
           .data(function (d) {
             return d.labels;
           });
-        labels.enter().append("text");
         transitions.labels(labels, labelAttributer);
+        labels.enter()
+          .append("text")
+          .call(labelAttributer)
+          .call(styliseur);
```

Comment:
I don't understand your first comment. I don't know if there is a bug. Do you mean in graph-vis-d3-js or in D3? Did you try e.g. http://graphviz.it/#/gallery/mike.gv and compare with my PR without the last commit. I thinks it's not an acceptable behavior.

Regarding your second comment; It seems that it is necessary to set the initial attributes for the enter selection directly before transition of the higher level elements starts. That's why I did it this way and applied the transition to the update selection only. I will investigate if it is possible to apply the transition to a merged selection and get rid of the styliseur dependency.